pkg/render/node.go

```diff
@@ -66,6 +67,24 @@ var (
 	nodeBGPReporterPort int32 = 9900
 
 	NodeTLSSecretName = "node-certs"
+
+	// calico-node DaemonSet container names used for validation.
+	// These should match CalicoNodeContainer.Name validation on the CalicoNodeDaemonSet type.
+	CalicoNodeDaemonSetContainerNames = map[string]struct{}{
```

Comment:
Doesn't look like this map is ever used (nor the one below)

Reply:
Removed 71f99a5